conf/drupal/config/block.block.alertsblock_2.yml

```diff
@@ -0,0 +1,21 @@
+uuid: 266bd473-0b53-4f17-8261-9a173b9ba390
+langcode: en
+status: true
+dependencies:
+  module:
+    - mass_alerts
+  theme:
+    - mass_theme
+id: alertsblock_2
+theme: mass_theme
+region: header
+weight: 0
+provider: null
+plugin: mass_alerts_block
+settings:
+  id: mass_alerts_block
+  label: 'Alerts block'
```

Comment:
Ideally these blocks have more descriptive name and label. Distinguish between sitewide and page-specific.